transports/janus_rabbitmq.c

```diff
@@ -290,6 +305,13 @@ int janus_rabbitmq_init(janus_transport_callbacks *callback, const char *config_
 			ssl_verify_hostname = TRUE;
 	}
 
+	/* Heartbeat config */
+	item = janus_config_get(config, config_general, janus_config_type_item, "heartbeat");
+	if(item && item->value && janus_string_to_uint16(item->value, &heartbeat) < 0) {
+		JANUS_LOG(LOG_ERR, "Invalid heartbeat timeout (%s), falling back to default\n", item->value);
+		heartbeat = 0;
```

Comment:
So the default is `0`. Does it mean that heartbeat is being disabled ?

Reply:
Yes - I've updated the log output